packages/base/server.js

```diff
@@ -48,107 +48,103 @@ module.exports.run = (executableSchema, middlewares, t, createGraphqlContext) =>
     })
     : null
 
-  return PgDb.connect().then(async (_pgdb) => {
-    pgdb = _pgdb
-    server = express()
-    httpServer = createServer(server)
-
-    // prod only
-    if (!DEV) {
-      // enable compression
-      server.use(compression())
-      // trust first proxy
-      server.enable('trust proxy')
-      // redirect to https
-      server.use((req, res, next) => {
-        if (!req.secure && (!IGNORE_SSL_HOSTNAME || req.hostname !== IGNORE_SSL_HOSTNAME)) {
-          res.redirect(`https://${req.hostname}${req.url}`)
-        }
-        return next()
-      })
-      // clear obsolete cf cookie
-      // this code was added 2018.01.19 - remove it after 2018.03.02
-      server.use((req, res, next) => {
-        res.clearCookie('__cfduid', { path: '/', httpOnly: true, domain: '.republik.ch' })
-        next()
-      })
-    }
-
-    // add req._log()
-    server.use(requestLog)
-
-    // monitor timeouts
-    if (REQ_TIMEOUT) {
-      server.use(
-        timeout(REQ_TIMEOUT, { respond: false }),
-        (req, res, next) => {
-          req.on('timeout', () => {
-            console.log('request timedout:', req._log())
-          })
-          next()
-        }
-      )
-    }
-
-    // Once DB is available, setup sessions and routes for authentication
-    auth.configure({
-      server: server,
-      secret: SESSION_SECRET,
-      domain: COOKIE_DOMAIN || undefined,
-      cookieName: COOKIE_NAME,
-      dev: DEV,
-      pgdb: pgdb
+  // connect to db
+  const _pgdb = await PgDb.connect()
+
+  pgdb = _pgdb
+  server = express()
+  httpServer = createServer(server)
+
+  // prod only
+  if (!DEV) {
+    // enable compression
+    server.use(compression())
+    // trust first proxy
+    server.enable('trust proxy')
+    // redirect to https
+    server.use((req, res, next) => {
+      if (!req.secure && (!IGNORE_SSL_HOSTNAME || req.hostname !== IGNORE_SSL_HOSTNAME)) {
+        res.redirect(`https://${req.hostname}${req.url}`)
+      }
+      return next()
+    })
+    // clear obsolete cf cookie
+    // this code was added 2018.01.19 - remove it after 2018.03.02
```

Comment:
Wondering if this comment is still true and if it should be removed indeed.

Might be a dedicated MR.

Reply:
Out of topic, but maybe a good moment to remove this! 👍